File: 0412. Fizz Buzz/fizz_buzz.py
from typing import List


class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ret = []
        for i in range(1, n + 1):
            s = ''
            if i % 3 == 0:
                s += 'Fizz'
            if i % 5 == 0:
                s += 'Buzz'
            if s == '':
                s = str(i)
            ret.append(s)
        return ret

File: 0412. Fizz Buzz/fizz_buzz.cpp
#include <string>
#include <vector>
using namespace std;

class Solution
{
public:
    vector<string> fizzBuzz(int n);
};

vector<string> Solution::fizzBuzz(int n)
{
    vector<string> ret;
    for (int i = 1; i <= n; i++) {
        string s = "";
        if (i % 3 == 0)
            s += "Fizz";
        if (i % 5 == 0)
            s += "Buzz";
        if (s.empty())
            s = to_string(i);
        ret.emplace_back(s);
    }
    return ret;
}
